jlm/rvsdg/node.hpp

```diff
@@ -73,6 +73,12 @@ class input
     return port_->type();
   }
 
+  inline const std::shared_ptr<const jlm::rvsdg::type> &
```

Comment:
I know it's implicitly inline by being in the class def. I still do it out of habit because it is intended to be inlined (cheap direct accessor to member variable).